### PR TITLE
Add community guidelines

### PR DESCRIPTION
**Pull request by switzersc-usds**
Update the repo with community guidelines shared and developed in our open source community chats.

### CODE REVIEW

**Review by esfoobar-usds — DISMISSED** [review]
🍏

File: COMMUNITY_GUIDELINES.md [hunk 1]
```diff
@@ -0,0 +1,33 @@
+# Justice40 Open Source Community Guidelines
+
+This document contains principles and guidelines for participating in the Justice40 open source community. 
+
+## Principles
+These principles guide our data, product, and process decisions, architecture, and approach. We discussed and developed these principles on the Open Source Community Kickoff on May 10, 2021. 
+
+* Open means transparent and participatory.
```
Comment (switzersc-usds) [inline COMMUNITY_GUIDELINES.md:8]: Hi! I think this is vague. What does this mean?

File: COMMUNITY_GUIDELINES.md [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+* We take a modular and modern approach to software development.
+* We build open source software and open source process.
```
Comment (switzersc-usds) [inline COMMUNITY_GUIDELINES.md:10]: Testing this out!

**Review by switzersc-usds — COMMENTED** [review]
I left a bunch of comments. Please let me know what you think!

File: COMMUNITY_GUIDELINES.md [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+* Fostering community includes building capacity and making our software and processes accessible to participants with diverse backgrounds and skill sets. 
+* Data (and data science) is as important as software and process. We build open data sets where possible.
+* We strive for transparency for algorithms and places we might be introducing bias.
+* We prioritize data sets that address community vulnerabilities for programs in Justice40.
+
+## Community Guidelines
+
+All community members are expected to adhere to our [Code of Conduct](CODE_OF_CONDUCT.md).
+
+Information on contributing to this repository aree available in our [Contributing file](CONTRIBUTING.md).
```
Comment (switzersc-usds) [inline COMMUNITY_GUIDELINES.md:21]: Typo! Aree

File: COMMUNITY_GUIDELINES.md [hunk 4]
```diff
@@ -0,0 +1,33 @@
...
+
+When participating in Justice40 open source community conversations and spaces, we ask individuals to follow the following guidelines:
+
+* When joining a conversation for the first time, please introduce yourself by providing a brief intro that includes:
+    * your related organization (if applicable)
+    * your pronouns
+    * disclosure of any current or potential financial interest in this work
```
Comment (switzersc-usds) [inline COMMUNITY_GUIDELINES.md:28]: Add superpower!

**Review by ginabeena — DISMISSED** [review]
Looks good to me 👍

**Pull request merged**